### DISCUSSION TITLE
question on internationalization of gorouter.dev

### DISCUSSION THREAD

**Discussion by toshi-kuji** [csells/go_router#198] [Q&A]
Hi Chris,
Thank you so much for creating this great library and the detailed documents.

I was wondering if the new gorouter.dev website has any plan to implement an internationalization feature.
If so, I would love to contribute to this project by submitting a translated version of the markdowns(to Japanese).

Are you expecting to have anything like this?

**Comment by csells** [comment]
I would happily accept a Japanese version of the docs! If you wanted to create a sub-folder under the docs folder with Japanese docs, then we can link them in.

**Reply by toshi-kuji** [reply]
Cool! Then, I will start working on the project on my weekends and create a PR. I think I can finish it within a couple of weeks. I promise you that I will keep the document up-to-date as much as I can! 😄
